packages/deploy/README.md

```diff
@@ -105,6 +111,45 @@ const deploymentId = '8181d9e0-88ce-4db0-802a-2b56e2e6a7b1';
 await client.Deployment.get(deploymentId);
 ```
 
+You can also retrieve the deploy approval process for a given network, which will return a `ApprovalProcessResponse` object
+
+```js
+await client.Deployment.getApprovalProcess('goerli');
+```
+
+### Upgrade
+
+To upgrade a contract you need to provide these required fields:
+
+- `proxyAddress`
+- `newImplementationAddress`
+- `network`
+
+There are a number of optional fields, these include:
+
+- `senderAddress` - The address you wish to create the Gnosis proposal with
```

Comment:
I would put this one last with some more explanations on how gnosis safe would fit it all of a sudden